src/MPQEditor/MPQEditor.ts

```diff
@@ -48,6 +50,24 @@ export class MPQEditorProvider implements vscode.CustomTextEditorProvider {
     );
   }
 
+  /**
+   * Whether mpqName is valid to to use
+   */
+  public static validateMPQName(
```

Comment:
this method seems to return `undefined` if name is valid.
1/ this seems unusual practice to me. can you provide where I can reference this usage? 
2/ please add doxygen format comment about this.

Reply:
> 1/ this seems unusual practice to me. can you provide where I can reference this usage?

See https://github.com/Samsung/ONE-vscode/pull/1511/files#r1156742956.
Also see https://github.com/Samsung/ONE-vscode/blob/4886fbbc782326d7d2c2f78547fcfe6133ed1c58/src/View/PasswordQuickInput.ts#L44
and https://github.com/Samsung/ONE-vscode/blob/0ad68fe538365a76952c2c866c70611f32738a17/src/OneExplorer/OneExplorer.ts#L769.

Reply:
> 2/ please add doxygen format comment about this.

ok.

Reply:
>  can you provide where I can reference this usage?

Not the caller, I want to see other programmers using this type from outside this repo or Samsung.

Reply:
> > can you provide where I can reference this usage?
> 
> Not the caller, I want to see other programmers using this type from outside this repo or Samsung.

May be this one:

https://github.com/microsoft/vscode-azure-blockchain-ethereum/blob/1107a87ad0efe300fd3b7080d787b52489548630/src/Models/TreeItems/InfuraNetworkNode.ts#L63

Reply:
my first comment
> this method seems to return undefined if name is valid.
1/ this seems unusual practice to me. can you provide where I can reference this usage?
2/ please add doxygen format comment about this.

is about return value that returns `undefined` for valid case, which would be better to be explained in the comment.

Reply:
Ahhh. Sorry. Ok.

Reply:
> is about return value that returns `undefined` for valid case, which would be better to be explained in the comment.

@seanshpark 
comments were added.

Reply:
it would be better to use https://www.doxygen.nl/manual/commands.html#cmdreturn